build.gradle.kts

```diff
@@ -17,11 +17,17 @@ subprojects {
         val ktlintVersion = "1.3.1"
         kotlin {
             target("**/*.kt")
-            targetExclude("**/build/**/*.kt")
+            targetExclude(
+                "**/build/**/*.kt",
+                // To be removed after the KMP Migration...
+                "src/commonMain/kotlin/com/fbada006/shared/data/ArtMakerSharedPreferences.kt",
```

Comment:
Add these as issues and assign yourself @emmanuelmuturia .

Reply:
Done...